.travis.yml

```diff
@@ -21,6 +21,8 @@ matrix:
     - os: linux
       sudo: required
       dist: trusty
+before_script:
+  - ./script/travis-bot || true
```

Comment:
I might be wrong but I think I moved this to the matrix to avoid doing this in every job. swiftlint doesn't really change when you do it for iOS/macOS/Linux so should be fine to do it in one job.
e.g. https://github.com/devlucky/Kakapo/blob/master/.travis.yml#L20